### PR TITLE
doc: update 1.6 upgrade guide

### PR DESCRIPTION
**Pull request by AlinsRan**
### Type of change:

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: docs/en/latest/upgrade.md [hunk 1]
```diff
@@ -21,9 +21,30 @@ title: Upgrade Guide
 #
 -->
 
+## Version change
+
+### ***1.6.0***
+
+The APISIX 3.x.x has adjusted the admin-api, in order to make APISIX Ingress compatible with these changes, You need to select the corresponding chart version, please refer [1.5 to 1.6](#15-to-16).
```
Comment (tao12345666333) [inline docs/en/latest/upgrade.md:28]:
```suggestion
The APISIX 3.x.x has changed the admin-api, in order to make APISIX Ingress compatible with these changes, You need to select the corresponding chart version, please refer [1.5 to 1.6](#15-to-16).
```

Comment (navendu-pottekkat) [inline docs/en/latest/upgrade.md:28]:
```suggestion
APISIX 3.x.x has changed the Admin API. To make APISIX Ingress compatible with these changes, you need to select the corresponding chart version. Please refer [1.5 to 1.6](#15-to-16).
```

File: docs/en/latest/upgrade.md [hunk 2]
```diff
@@ -21,9 +21,30 @@ title: Upgrade Guide
...
+
+### ***1.5.0***
+
+- CRD has been upgraded to the V2 version, and V2beta3 has been marked as deprecated.
+- `app_namespace` is deprecated, you can use `namespace_selector` instead.
```
Comment (navendu-pottekkat) [inline docs/en/latest/upgrade.md:33]:
```suggestion
- `app_namespace` is deprecated,. You can use `namespace_selector` instead.
```

File: docs/en/latest/upgrade.md [hunk 3]
```diff
@@ -64,23 +85,28 @@ helm upgrade apisix apisix/apisix \
...
-
 ### Compatible upgrade
 
 Compatible upgrades can be made without changing any resources.
 
 #### ***1.5 to 1.6***
 
+[Relevant version information and compatibility of apisix-helm-chart](https://github.com/apache/apisix-helm-chart#compatibility-matrix).
+
+You need to select the corresponding chart version according to the APISIX version to install or upgrade. As shown in the following table:
```
Comment (navendu-pottekkat) [inline docs/en/latest/upgrade.md:96]:
```suggestion
You need to select the corresponding chart version according to the APISIX version as shown to install or upgrade:
```

**Review by navendu-pottekkat — COMMENTED** [review]
Others LGTM.

**Review by navendu-pottekkat — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Pull request merged**